MARU/MARU/Screens/Main/ViewModel/ResultSearchViewModel.swift

```diff
@@ -53,10 +60,21 @@ final class ResultSearchViewModel: ViewModelType {
     let reSearch = input.tapTextField
       .asDriver()
 
+    let canMakeGroup = canMakeGroupPublishSubject
+      .flatMap(NetworkService.shared.auth.myGroup)
+      .map { $0.data?.myGroups.count ?? -1 }
+      .map { count in
+        if count >= 0 && count < 2 {
+          return  true
+        } else { return false }
```

Comment:
```suggestion
        if count >= 0 && count < 2 {
          return  true
        } 
        return false
```
제가 원한 방식은 이런 방식이었습니다~

리뷰 바로 리졸브 하지는 말아주세요~ 호호...